debian-live/build.sh

```diff
@@ -168,6 +174,18 @@ function configHooks {
     logerror "${FUNCNAME[0]}" "Debian Live envsubst for flatpak failed"
     exit 1
   fi
+
+  GNOME_SHELL_EXTENSIONS="$(yq '.packages.gnome-shell | select(.enable) | .extension | filter(.enable) | map (.id) | join (" ")' "${WORK_DIR}"/debian-live/package.yaml)"
```

Comment:
please add `GNOME_SHELL_EXTENSIONS` to https://github.com/batchworksde/iksdp_desktop/blob/63a980df838542660cf5b10da83796f384397783/debian-live/build.sh#L158